### DISCUSSION TITLE
TENYEARTIMELOCK on Masternode

### DISCUSSION THREAD

**Discussion by supervolot** [DeFiCh/app#1042] [Q&A]
Hi, i get the fail error "Expected type array, got string" when i try to make a new masternode with 

createmasternode "ownerAdress" "operatorAdress" "TENYEARTIMELOCK"

Has anyone the correct texting for me

thx

**Comment by Stonygan** [comment]
`createmasternode ownerAdress operatorAdress [] TENYEARTIMELOCK`
